torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -319,6 +319,7 @@ void AliasDb::analyzeImpl(Node* node) {
       return analyzeLoop(node);
     case prim::FusionGroup:
     case prim::DifferentiableGraph:
+    case prim::FunctionalGraph:
```

Comment:
do we need to analyze the entire subgraph? I think we can just treat it as a purely functional node.

Reply:
I'll do this in a follow up when i add the implicit copies.